src/components/MovieInfo/MovieInfo.js

```diff
@@ -1,28 +1,39 @@
 
 import React, { Component } from 'react';
-import testDetails from '../MovieInfo/mockMovieView';
+import { fetchSingleMovieDetails } from '../../apiCalls';
 import './MovieInfo.css';
 
 class MovieInfo extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      movie: testDetails.movie
-    };
+  constructor() {
+    super();
+    this.state = {};
```

Comment:
I'd love to hear the reasoning behind dropping the `movie` property in state here.    What benefit have you found from doing this?  Any drawbacks?